vale/CockroachDB/LocalizedLinks.yml

```diff
@@ -0,0 +1,8 @@
+extends: existence
+message: "Link contains a localization element (e.g., `en`, `en-us`). Try to remove it from the URL: '%s'"
+link: https://github.com/cockroachdb/docs/blob/main/StyleGuide.md#links
+scope: raw
+level: warning
+nonword: true
+tokens:
+  - 'https:\/\/.*?(\ben\b).*?(?="|\| ))'
```

Comment:
I think this is too broad -- it would trigger on `/enticing/` for example. I think we need to make the test more specific. The main variants I have seen are `/en/`, `/en-us/`, `/en_us/`, and the same strings as `GET` parameters. WDYT?

Reply:
Fixed.